providers/os/resources/os.lr

```diff
@@ -616,6 +616,9 @@ package @defaults("name version") {
 
   // Package origin (optional)
   origin() string
+  // Package origin version (optional)
+  originVersion() string
```

Comment:
should this not be origin package that references another full package? We have the following options:

```
// dict returns name and version field
originPackage() dict

// return full package resource
originPackage() package
```

Reply:
As discuessed, I now include the version in the origin field as it is shown in dpkg output:
```
samba (2:4.17.12+dfsg-0+deb12u1)
```